diff --git a/tests/conftest.py b/tests/conftest.py
index a2d910c694..6d65c433b8 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -902,12 +902,11 @@ class HfRunner:
             # where vllm worker processes are still alive and holding GPU
             # memory when hf_runner.__exit__ is called.
             from tests.utils import (
-                get_physical_device_indices,
                 record_gpu_memory_usage_stats,
             )
 
             if (device_count := current_platform.device_count()) > 0:
-                devices = get_physical_device_indices(devices=list(range(device_count)))
+                devices = list(range(device_count))
                 mem_usage_stats = record_gpu_memory_usage_stats(devices=devices)
                 self.threshold_ratios = {
                     device: 0.05 + mem_used / mem_tot
